File: java/src/main/java/com/android/frame/mvc/viewBinding/AATest/TestMvcActivity.java
package com.android.frame.mvc.viewBinding.AATest;

import android.os.Bundle;

import com.android.frame.mvc.viewBinding.BaseActivity_VB;
import com.android.java.databinding.ActivityCommonLayoutBinding;
import com.android.util.CommonLayoutUtil;

/**
 * Created by xuzhb on 2020/7/23
 * Desc:
 */
public class TestMvcActivity extends BaseActivity_VB<ActivityCommonLayoutBinding> {

    @Override
    public void handleView(Bundle savedInstanceState) {
        CommonLayoutUtil.initCommonLayout(this, "MVC框架", "天气信息", "网易新闻");
    }

    @Override
    public void initListener() {
        binding.btn1.setOnClickListener(v -> startActivity(WeatherActivity.class));
        binding.btn2.setOnClickListener(v -> startActivity(NewsListActivity.class));
    }

    @Override
    public ActivityCommonLayoutBinding getViewBinding() {
        return ActivityCommonLayoutBinding.inflate(getLayoutInflater());
    }

}

File: kotlin/src/main/java/com/android/frame/mvc/viewBinding/AATest/TestMvcActivity.kt
package com.android.frame.mvc.viewBinding.AATest

import android.os.Bundle
import com.android.basicproject.databinding.ActivityCommonLayoutBinding
import com.android.frame.mvc.viewBinding.BaseActivity_VB
import com.android.util.initCommonLayout

/**
 * Created by xuzhb on 2020/7/23
 * Desc:
 */
class TestMvcActivity : BaseActivity_VB<ActivityCommonLayoutBinding>() {

    override fun handleView(savedInstanceState: Bundle?) {
        initCommonLayout(this, "MVC框架", "天气信息", "网易新闻")
    }

    override fun initListener() {
        binding.btn1.setOnClickListener {
            startActivity(WeatherActivity::class.java)
        }
        binding.btn2.setOnClickListener {
            startActivity(NewsListActivity::class.java)
        }
    }

    override fun getViewBinding() = ActivityCommonLayoutBinding.inflate(layoutInflater)

}